### PR TITLE
added zeru

### PR DESCRIPTION
**Pull request by elitex45**
## Gauge List Contribution

Please make sure you've completed the following before submitting your pull request:

- [x] I have read the [contribution guidelines](README.md)

## Summary by CodeRabbit

- **New Features**
	- Introduced two new gauges: "zberaHONEY" and "ZERU-HONEY," enhancing the gauge offerings.
	- Added a new protocol entry for "Zeru," providing users with additional information and resources.
	- Added two new tokens: "zberaHONEY" and "ZERU," expanding the available token options.

These updates improve user access to diverse gauge options, relevant protocol details, and additional token choices.

### CODE REVIEW

File: src/.DS_Store
Comment (mlenser) [inline src/.DS_Store:1]: Hey @elitex45 thanks for contributing! I tried to manually delete these on your fork, but I can't manage. Could you either:
- Delete these files from git
- Recreate this branch now that we gitignore these files

(whichever is easier)

File: src/assets/tokens/zberaHONEY.svg
Comment (mlenser) [inline src/assets/tokens/zberaHONEY.svg:1]: Please add this token to the `defaultTokenList.json`

File: src/assets/tokens/zeru.svg
Comment (mlenser) [inline src/assets/tokens/zeru.svg:1]: Please add this token to the `defaultTokenList.json`

File: src/gauges/bartio/defaultGaugeList.json [hunk 1]
```diff
@@ -134,6 +134,27 @@
...
+    },
+    {
+      "beraRewardsVault": "0x3E1FfDD1AD1e4774693CBCCdF9C737De8713D984",
+      "lpTokenAddress": "0x12Afd7A3324B689e32eEa71902DCbA5ED72Ee67E",
+      "mintUrl": "https://testnet.zeru.finance/v1/Assets?asset=0x0e4aaf1351de4c0264c5c7056ef3777b41bd8e03",
+      "name": "zberaHONEY",
+      "protocol": "zeru",
+      "types": ["vault"],
+      "underlyingTokens": ["0x0E4aaF1351de4c0264C5c7056Ef3777b41BD8e03"]
+    },
```
[the remark this answers is not in the document]
Comment (elitex45) [reply src/gauges/bartio/defaultGaugeList.json:146]: Berachain is not real yet XD

File: src/.DS_Store
Comment (elitex45) [reply src/.DS_Store:1]: Added to the .gitignore

File: src/assets/tokens/zberaHONEY.svg
Comment (elitex45) [reply src/assets/tokens/zberaHONEY.svg:1]: Done

File: src/assets/tokens/zeru.svg
Comment (elitex45) [reply src/assets/tokens/zeru.svg:1]: Done

File: .gitignore
Comment (mlenser) [inline .gitignore:1]: Please revert this - the .DS_STORE at the root level should catch others.

Comment (elitex45) [reply .gitignore:1]: Done

**Comment by mlenser** [comment]
Thanks @elitex45. I added zeru via https://github.com/infrared-dao/infrared-default-list/pull/15. Apologies for doing it as its own PR, but I wanted to avoid the DS_STORE in the git history and the changes to the `.gitignore` file.

We'll work to improve this process.

We aren't using the data from this repository quite yet. Our new backend is almost done which will allow us to do so. Please allow at least a few days, but maybe a week or two. Thank you for your patience!

**Comment by elitex45** [comment]
Thanks!

**Pull request closed without merging**